packages/flutter/lib/src/widgets/app.dart

```diff
@@ -232,7 +232,7 @@ Locale basicLocaleListResolution(
   // supported locale.
   final Locale resolvedLocale = matchesLanguageCode ?? matchesCountryCode ?? supportedLocales.first;
   return resolvedLocale;
-}
+      }
```

Comment:
FYI there are weird formatting changes that shouldn't have been made here and below.